core/node/eth_watch/src/tests.rs

```diff
@@ -116,15 +145,19 @@ impl EthClient for MockEthClient {
             if let Some(ops) = self.inner.read().await.diamond_upgrades.get(&number) {
                 logs.extend_from_slice(ops);
             }
-            if let Some(ops) = self.inner.read().await.governance_upgrades.get(&number) {
+            if let Some(ops) = self.inner.read().await.upgrade_timestamp.get(&number) {
                 logs.extend_from_slice(ops);
             }
         }
-        Ok(logs)
+        Ok(logs
+            .into_iter()
+            .filter(|log| {
+                log.topics.contains(&topic1)
+                    && (topic2.is_none() || log.topics.contains(&topic2.unwrap()))
```

Comment:
nit: contains is incorrect here as e.g. forth topic is `log.topics` can be equal to `topic1`